src/Microsoft.AspNet.Server.Kestrel/KestrelEngine.cs

```diff
@@ -95,6 +101,7 @@ public IDisposable CreateServer(string scheme, string host, int port, Func<Frame
             {
                 var listener = new Listener(Memory);
                 listener.StartAsync(scheme, host, port, thread, application).Wait();
+                Logger.WriteInformation(string.Format("Server listening on {0}://{1}:{2}", scheme, host, port));
```

Comment:
This console output is a lie, because [the `host` parameter is not used by the listener](https://github.com/aspnet/KestrelHttpServer/blob/dev/src/Microsoft.AspNet.Server.Kestrel/Http/Listener.cs#L58). In the current implementation the server is always listening on `IPAddress.Any`.